lib/components/app/call-taker-panel.js

```diff
@@ -251,15 +251,8 @@ class CallTakerPanel extends Component {
           <div className='desktop-narrative-container'>
             <NarrativeItineraries
               containerStyle={{
-                bottom: '0',
                 display: 'flex',
-                flexDirection: 'column',
-                left: '0',
-                position: 'absolute',
-                right: '0',
-                // FIXME: This top pixel value can be variable dependent on
-                // height of the form above. It may need to be specified differently
-                top: 210 + intermediatePlaces.length * 45
+                flexDirection: 'column'
```

Comment:
One side effect of this change is that the results are no longer on the edge of the panel:
![image](https://user-images.githubusercontent.com/56846598/142239212-87d3b939-e7da-4b1a-9a5b-cf2b7773dbf6.png)